src/foremast/awslambda/awslambda.py

```diff
@@ -227,7 +228,13 @@ def create_function(self, vpc_config):
                 Timeout=int(self.timeout),
                 MemorySize=int(self.memory),
                 Publish=False,
-                VpcConfig=vpc_config)
+                VpcConfig=vpc_config,
+                Tags={
+                    'app_group': self.group,
+                    'app_name': self.repo,
```

Comment:
It looks like that we (Gogo) use the full app_name for the `app_name` tag, not just the repo name. Lets keep this consistent with EC2 tags.